### PR TITLE
fixed the gap between note/title placeholder

### LINKED ISSUE #7070: Gap Between Note Title Input and Placeholder
# Bug Description

There is a gap between the note title input and the note/task title placeholder.

CleanShot 2024-09-16 at 18 32 16@2x

# Desired Behavior

The note/task title input should appear above the note/task title placeholder, not beside it.

### PR DESCRIPTION
**Pull request by yuvaraj8019**
closes #7070

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/field/input/components/TextAreaInput.tsx [hunk 1]
```diff
@@ -36,12 +36,17 @@ const StyledTextArea = styled(TextareaAutosize)`
...
 const StyledTextAreaContainer = styled.div`
-  border: ${({ theme }) => `1px solid ${theme.border.color.medium}`};
+  border: ${({ theme }) => `1px solid ${theme.border.color.light}`};
   position: relative;
   width: 100%;
-  padding: ${({ theme }) => theme.spacing(2)} ${({ theme }) => theme.spacing(0)};
+  padding: ${({ theme }) => theme.spacing(2)} ${({ theme }) => theme.spacing(1)};
   border-radius: ${({ theme }) => theme.border.radius.sm};
   background: ${({ theme }) => theme.background.primary};
+  @media (max-width: 768px) {
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/ui/field/input/components/TextAreaInput.tsx:45]: this does not feel right! Could you share more about what you are trying to achieve? Issue link or issue screenshot?

Comment (yuvaraj8019) [reply packages/twenty-front/src/modules/ui/field/input/components/TextAreaInput.tsx:45]: issue link added in the description  

- if the screen size width is less than 768px a gap between placeholder and input field occurs
- screenshot before update
367842746-7b447195-cbc0-4f0c-b708-9c3808f05975

- added a media query to fix the gap

-screenshot after update
Screenshot 2024-10-13 at 10 05 53 PM

**Comment by charlesBochet** [comment]
Thanks for linking the issue, I believe it has already been solved in a previous PR. Can you still see the issue on main branch?

**Comment by charlesBochet** [comment]
I have double checked and the issue seems to be fixed, closing this PR and the issue.
Sorry about that @yuvaraj8019, feel free to work on any other good first issue

**Pull request closed without merging**